### ISSUE TITLE
Contao 5.x and symfony 6.x

### ISSUE THREAD

**Issue by sdahlmann** [pdir/content-migration-bundle#5]
Do you plan to adapt the extension "pdir / content-migration-bundle" to Contao 5.x and symfony 6.x?
I would be very happy about an adjustment.

**Comment by MDevster** [comment]
Yes, we will, but it will still take a while, as we first have to update all the paid extensions.
